public/locales/en/translations.json

```diff
@@ -740,11 +744,17 @@
         "title": "Delete Package?",
         "info": "Deleting selected package will remove it. It will not impact any of the deployments of this package."
       }
-    },
-    "typeOptions": {
-      "edgemanifest": "Edge Manifest"
     }
   },
+  "packageTypes": {
+    "edgeManifest": "Edge Manifest",
+    "deviceConfiguration": "Device Configuration",
+    "unknown": "Unknown"
+  },
+  "configTypes": {
+    "firmware": "Firmware",
```

Comment:
Should this be "Firmware Update"?

Reply:
Yes according to PM spec and I double checked.